### PR TITLE
en, zh: move tidb-operator architecture section to a new page

### LINKED ISSUE #386: Move tidb-operator architecture section to a new page
Move https://pingcap.com/docs/tidb-in-kubernetes/stable/tidb-operator-overview/#tidb-operator-architecture to a new page under architecture section.

### PR DESCRIPTION
**Pull request by cofyc**
### What is changed, added or deleted? (Required)

fixes #386 
suggested by Liquan Pei and @DanielZhangQD 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/architecture.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: TiDB Architecture
```
Comment (ran-huang) [inline en/architecture.md:2]:
```suggestion
title: TiDB Operator Architecture
```

File: en/architecture.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+summary: Learn the architecture of tidb-operator and how it works.
```
Comment (ran-huang) [inline en/architecture.md:3]:
```suggestion
summary: Learn the architecture of TiDB Operator and how it works.
```

File: en/architecture.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+category: reference
+---
+
+# TiDB Operator Architecture
+
```
Comment (ran-huang) [inline en/architecture.md:8]:
```suggestion

This document describes the architecture of TiDB Operator and how it works.

```

File: en/architecture.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+![TiDB Operator Overview](/media/tidb-operator-overview-1.1.png)
```
Comment (ran-huang) [inline en/architecture.md:9]:
```suggestion
## Architecture

The following diagram is an overview of the architecture of TiDB Operator.

![TiDB Operator Overview](/media/tidb-operator-overview-1.1.png)
```

File: en/architecture.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* `tidb-scheduler` is a Kubernetes scheduler extension that injects the TiDB specific scheduling policies to the Kubernetes scheduler;
+* `tidb-admission-webhook` is a dynamic admission controller in Kubernetes, which completes the modification, verification, operation and maintenance of Pod, StatefulSet and other related resources.
+
+In addition, TiDB Operator also provides `tkctl`, the command-line interface for TiDB clusters in Kubernetes. It is used for cluster operations and troubleshooting cluster issues.
+
+![TiDB Operator Control Flow](/media/tidb-operator-control-flow-1.1.png)
+
+The diagram above is the analysis of the control flow of TiDB Operator. Starting from TiDB Operator v1.1, the TiDB cluster, monitoring, initialization, backup, and other components are deployed and managed using CR. The overall control flow is described as follows:
```
Comment (ran-huang) [inline en/architecture.md:31]:
```suggestion
## Control flow

The following diagram is the analysis of the control flow of TiDB Operator. Starting from TiDB Operator v1.1, the TiDB cluster, monitoring, initialization, backup, and other components are deployed and managed using CR. 

![TiDB Operator Control Flow](/media/tidb-operator-control-flow-1.1.png)

The overall control flow is described as follows:
```

File: zh/architecture.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: TiDB Operator 架构
+summary: 了解 TiDB Operator 架构以其工作原理。
```
Comment (ran-huang) [inline zh/architecture.md:3]:
```suggestion
summary: 了解 TiDB Operator 架构及其工作原理。
```

File: zh/architecture.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+category: reference
+---
+
+# TiDB Operator 架构
+
```
Comment (ran-huang) [inline zh/architecture.md:8]:
```suggestion

本文档介绍 TiDB Operator 的架构及其工作原理。

```

File: zh/architecture.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+![TiDB Operator Overview](/media/tidb-operator-overview-1.1.png)
```
Comment (ran-huang) [inline zh/architecture.md:9]:
```suggestion
## 架构

下图是 TiDB Operator 的架构概览。

![TiDB Operator Overview](/media/tidb-operator-overview-1.1.png)
```

File: zh/architecture.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+* `tidb-controller-manager` 是一组 Kubernetes 上的自定义控制器。这些控制器会不断对比 `TidbCluster` 对象中记录的期望状态与 TiDB 集群的实际状态，并调整 Kubernetes 中的资源以驱动 TiDB 集群满足期望状态，并根据其他 CR 完成相应的控制逻辑；
+* `tidb-scheduler` 是一个 Kubernetes 调度器扩展，它为 Kubernetes 调度器注入 TiDB 集群特有的调度逻辑。
+* `tidb-admission-webhook` 是一个 Kubernetes 动态准入控制器，完成 Pod、StatefulSet 等相关资源的修改、验证与运维。
+
+此外，TiDB Operator 还提供了命令行接口 `tkctl` 用于运维集群和诊断集群问题。
+
+![TiDB Operator Control Flow](/media/tidb-operator-control-flow-1.1.png)
+
+上图是 TiDB Operator 的控制流程解析。从 TiDB Operator v1.1 开始，TiDB 集群、监控、初始化、备份等组件，都通过 CR 进行部署、管理。整体的控制流程如下：
```
Comment (ran-huang) [inline zh/architecture.md:31]:
```suggestion
## 流程解析

下图是 TiDB Operator 的控制流程解析。从 TiDB Operator v1.1 开始，TiDB 集群、监控、初始化、备份等组件，都通过 CR 进行部署、管理。

![TiDB Operator Control Flow](/media/tidb-operator-control-flow-1.1.png)

整体的控制流程如下：
```

**Comment by DanielZhangQD** [comment]
@Ishiihara PTAL.

File: en/TOC.md [hunk 10]
```diff
@@ -58,6 +58,7 @@
 - [FAQs](faq.md)
 + Reference
   + Architecture
+    - [TiDB Operator](architecture.md)
```
Comment (DanielZhangQD) [inline en/TOC.md:61]: What about `Architecture(architecture.md)`?

Comment (ran-huang) [reply en/TOC.md:61]: `Architecture` repeats with the second level menu. 
How about `Overview`?

Comment (cofyc) [reply en/TOC.md:61]: `Architecture ` repeats the menu, I think `Overview` is better here. What's the corresponding Chinese version?

Comment (ran-huang) [reply en/TOC.md:61]: I propose `概览` for Chinese. Any suggestion?

Comment (DanielZhangQD) [reply en/TOC.md:61]: I mean just `Architecture(architecture.md)`, no need to add a new item, so when users click `Architecture`, they can see the introduction directly.

Comment (ran-huang) [reply en/TOC.md:61]: @cofyc Please change this to `overview` and edit the zh TOC accordingly~

**Pull request merged**